src/main/kotlin/no/nav/medlemskap/regler/v1/ReglerForLovvalg.kt

```diff
@@ -91,21 +96,23 @@ class ReglerForLovvalg(val datagrunnlag: Datagrunnlag) : Regler() {
         }
     }
 
-    private fun sjekkLandskode(): Resultat =
-            when {
-                sjekkBrukersPostadresseOgBostedsadresseLandskode() -> ja()
-                else -> nei("Bruker har enten ikke norsk bostedsadresse eller postadresse")
-            }
+    private fun sjekkLandkode(): Resultat {
+        val bostedsadresser = bostedsadresser.adresserSiste12Mnd(kontrollPeriodeForPersonhistorikk)
+        val postadresserLandkoder = postadresser.landkodeTilAdresseSiste12Mnd(kontrollPeriodeForPersonhistorikk)
+        val midlertidigadresserLandkoder = midlertidigAdresser.landkodeTilAdresseSiste12Mnd(kontrollPeriodeForPersonhistorikk)
+
+        return when {
+            bostedsadresser.erIkkeTom()
+                    && (postadresserLandkoder alleEr "NOR" || postadresserLandkoder.erTom())
```

Comment:
Bør unngå sjekk på magiske strengverdier som "NOR". Enten definere konstant for "NOR", eller definerere funksjon erNorskLandkode().